src/core/server/server.api.md

```diff
@@ -976,6 +977,27 @@ export interface OnPreAuthToolkit {
     rewriteUrl: (url: string) => OnPreAuthResult;
 }
 
+// @public
+export interface OnPreResponseExtensions {
+    headers?: ResponseHeaders;
+}
+
+// Warning: (ae-forgotten-export) The symbol "OnPreResponseResult" needs to be exported by the entry point index.d.ts

```

Comment:
Should this be a `@public` type and exported since it's a return type of a public API?

Reply:
It's for internal use only. Plugins should know that's an expected result, but not about its internals.